dask_jobqueue/pbs.py

```diff
@@ -83,15 +83,16 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
             header_lines.append('#PBS -q %s' % queue)
         if project is not None:
             header_lines.append('#PBS -A %s' % project)
-        if resource_spec is None:
+        if resource_spec is None and self.worker_processes and self.worker_threads:
```

Comment:
As discussed, I believe we should fail fast if processes and cores are not provided, so these checks could be removed. Nevertheless, they don't hurt, so we can keep them!